source/includes/artifacts/_031-get-fetch-artifact-metadata.md.erb

```diff
@@ -0,0 +1,33 @@
+## Get Fetch Artifact Metadata
+
+This is a message that the plugin should, to allow users to configure fetch artifact properties from the Fetch Artifact Task View in GoCD.
```

Comment:
`This is a message that the plugin should, to allow users to configure fetch artifact properties from the Fetch Artifact Task View in GoCD.` -> `This is a message that the plugin should implement, to allow users to configure fetch artifact task from the Task View in GoCD.`